jekyll/_cci2/orb-author-intro.md

```diff
@@ -57,6 +57,8 @@ Enter the following command to claim your namespace, if you have not yet claimed
 circleci namespace create <name> <vcs-type> <org-name> [flags]
 ```
 
+where `name` is the namespace you wish to claim, `vcs-type` is the type of your version control system (i.e. `github` or `bitbucket`), and `org-name` is the name of your organisation
```

Comment:
```suggestion
where `name` is the namespace you wish to claim, `vcs-type` is the type of your version control system (i.e. `github` or `bitbucket`), and `org-name` is the name of your organization.
```

Reply:
Good call. We should have this information here. I have made one small suggestion which you can commit using GitHub's interface; then we will be all set to merge.

Thank you for contributing!